versions.txt

```diff
@@ -7,7 +7,7 @@ docker_ubuntu_version=16.04
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
 clear_vm_image_version=14700
-clear_vm_kernel_version=4.9.24-60
+clear_vm_kernel_version=4.9.27-63
```

Comment:
Please remove the `-6x` part we do not need it it's only the counter of builds for each packaging.